library/Terminal42/ChangeLanguage/EventListener/DataContainer/PageFieldsListener.php

```diff
@@ -82,8 +82,23 @@ public function onSaveLanguageMain($value, DataContainer $dc)
                 [$value, $dc->id]
             );
 
-            if ($duplicates > 0) {
-                throw new \RuntimeException($GLOBALS['TL_LANG']['MSC']['duplicateMainLanguage']);
+
+            if (count($duplicates) > 0) {
+
+                $properties = [];
+
+                foreach ($duplicates as $duplicate) {
+                    $properties[] = [
```

Comment:
Why create this array and not just `sprintf` the label here?

Reply:
Totally agree, thanks for the hint!